meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
as far as I remember, the boot cmdline is just to collect the stuff. You
still need to manually ask for the svg to be rendered, but I may be too old
fashioned :-)

On Fri, Jun 10, 2016 at 11:18 AM, Geoffroy Van Cutsem <
notifications@github.com> wrote:

> In meta-ostro/conf/local.conf.sample
> <https://github.com/ostroproject/meta-ostro/pull/204#discussion_r66620359>
> :
>
> > @@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
> >  #
> >  #require conf/distro/include/ostro-os-production.inc
> >
> > +# systemd-bootchart is a useful tool to analyze and optimize a system
> > +# boot time. The tool is available in Ostro and needs to be activated
> > +# by a kernel command-line parameter which requires to build a new
> > +# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
> > +# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
> > +# is not enabled by default. For instructions on how to modify an Ostro
> > +# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
>
> It's installed in /lib in Ostro, not /usr/lib
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ostroproject/meta-ostro/pull/204/files/918986fe9b4e4ec9d104c5fcd93e8c0e5db7172d#r66620359>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAHLqJeh6FgAb6b78-63lkLzAM5UqQrks5qKXIcgaJpZM4Iw_6M>
> .
>



-- 
Gustavo Sverzut Barbieri
--------------------------------------
Mobile: +55 (19) 99225-2202
Contact: http://www.gustavobarbieri.com.br/contact